### PR TITLE
rtmp-services: Update Switchboard Live servers

### PR DESCRIPTION
**Pull request by are0h**
### Description
The Switchboard Live server URLs have been updated in the application, so the URLs need to updated in OBS to match.

### Motivation and Context
There were recent changes to the various ingestion servers that Switch

### How Has This Been Tested?

This was tested using the latest build of OBS on macOS 12.3.1. 
Testing consisted of going live with every server to ensure connectivity was stable. 

### Types of changes

 - Tweak (non-breaking change to improve existing functionality) 

### Checklist:

- [ ] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

File: plugins/rtmp-services/data/package.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-    ]
+	"$schema": "schema/package-schema.json",
+	"url": "https://obsproject.com/obs2_update/rtmp-services",
+	"version": 205,
+	"files": [
+		{
+			"name": "services.json",
+			"version": 205
+		}
+	]
```
Comment (notr1ch) [inline plugins/rtmp-services/data/package.json:10]: Please do not make unnecessary whitespace modifications.

File: plugins/rtmp-services/data/services.json [hunk 2]
```diff
@@ -1143,66 +1143,51 @@
...
             ],
             "recommended": {
                 "keyint": 2,
-                "max audio bitrate": 128,
-                "max video bitrate": 10000
-            }
+                "profile": "main",
+                "max audio bitrate": 160,
+                "max video bitrate": 6000
+                }
```
Comment (notr1ch) [inline plugins/rtmp-services/data/services.json:1190]: Indentation level is incorrect..

File: plugins/rtmp-services/data/services.json [hunk 3]
```diff
@@ -1143,66 +1143,51 @@
...
+                    "name": "UK",
+                    "url": "rtmp://ingest-uk.switchboard.zone/live"
                 }
             ],
             "recommended": {
                 "keyint": 2,
-                "max audio bitrate": 128,
-                "max video bitrate": 10000
-            }
+                "profile": "main",
```
Comment (notr1ch) [inline plugins/rtmp-services/data/services.json:1187]: Are you sure this is wanted? It will reduce quality for compatibility with devices that are 10+ years old.

**Review by notr1ch — CHANGES_REQUESTED** [review]

File: plugins/rtmp-services/data/package.json (hunk 1 above)
Comment (are0h) [reply plugins/rtmp-services/data/package.json:10]: Ooop, editor setting. My bad.

File: plugins/rtmp-services/data/services.json (hunk 3 above)
Comment (are0h) [reply plugins/rtmp-services/data/services.json:1187]: What is the recommended setting for max compatibility?
